trino-aws-proxy/src/test/java/io/trino/aws/proxy/server/TestOpaSecurity.java

```diff
@@ -61,7 +62,7 @@ public static class Filter
         public TestingTrinoAwsProxyServer.Builder filter(TestingTrinoAwsProxyServer.Builder builder)
         {
             return builder.withProperty("s3-security.type", OPA_S3_SECURITY_IDENTIFIER)
-                    .withProperty("opa-s3-security.server-base-uri", "http://localhost")
+                    .withProperty("opa-s3-security.server-base-uri", "http://localhost:%s/v1/data".formatted(OPA_CONTAINER_PORT))
```

Comment:
This is not guaranteed to be 8181, 8181 is the port in the OPA container.

I think we need to keep the port resolution in the way it was implemented before